src/tooltool/client/tooltool.py

```diff
@@ -43,10 +41,10 @@
 import tempfile
 import threading
 import time
-import urllib2
-import urlparse
 import zipfile
 
+from io import open
+from io import BytesIO as StringIO
```

Comment:
Question: I'm confused by this alias. Why is it needed for? I find it a little bit misleading since it's named after an other existing class.

Reply:
right, i think this is something that was left over when i was trying to figure out StringIO in python 3

Reply:
done